storage/simulation/cluster.go

```diff
@@ -169,15 +177,20 @@ func (c *Cluster) splitRange(rangeID roachpb.RangeID) {
 // runEpoch steps through a single instance of the simulator. Returns true if no
 // actions were performed during this epoch.
 // Each epoch performs the following steps:
-// 1) The status of every store is gossiped so the store pool is up to date.
-// 2) Each replica on every range calls the allocator to determine if there are
+// 1) Perform all scripted actions.
```

Comment:
this is an imperative voice, while the existing items are passive voices (except for "each replica ...", which is also inconsistent). Can you homogenize all of these?